mesonbuild/interpreter/dependencyfallbacks.py

```diff
@@ -270,7 +270,7 @@ def _handle_featurenew_dependencies(self, name: str) -> None:
             FeatureNew.single_use('OpenMP Dependency', '0.46.0', self.subproject)
 
     def _notfound_dependency(self) -> NotFoundDependency:
-        return NotFoundDependency(self.environment)
+        return NotFoundDependency(' '.join(self.names), self.environment)
```

Comment:
Just take `self.names[0]`, that list is fallbacks when a dependency could have multiple names.

Reply:
Done